content/topics/sharing-code-workflows.md

```diff
@@ -0,0 +1,84 @@
+
+* **Transparency and Reproducibility** – Clear documentation and the ability to replicate analyses build trust in scientific results, and is essential for verifying and understanding research findings.
+* **Reusability and Preservation** – Sharing enables workflows to be applied to new datasets or challenges, maximising the utility and lifespan of research tools and preventing the loss of valuable methodologies.
+
+### Enhance research efficiency and compliance
+
+* **Operational Efficiency** – Modular, well-documented software facilitates smoother research processes, from rerunning experiments to adapting methodologies for new data. 
+* **Legal and Ethical Compliance** – Separate publication of software and data simplifies adherence to licensing requirements, while meeting journals' and funders' open code mandates strengthens research credibility and dissemination.
+* **Recognition** – Open code obtains scholarly recognition through citations, emphasizing the importance of software as a research output.
+
+### Foster collaboration and global innovation
+* **Global Access and Education** – Sharing tools broadens their reach, enabling equitable access and serving as vital resources for training the next generation of scientists, particularly in resource-limited settings.
+* **Community Contributions and Cross-Field Innovation** – Open software encourages community input and interdisciplinary exchanges, leading to more robust, flexible solutions and sparking innovation across research domains.
+
+<a class="link-teal" href="https://www.software.ac.uk/blog/why-should-you-care-about-reproducible-code-and-how-get-started" target="_blank"> <b>Learn more about practical benefits and how to get started <i class="bi bi-box-arrow-up-right"></i></b></a>
+
+<a class="link-teal" href="https://urn.kb.se/resolve?urn=urn:nbn:se:kb:publ-738" target="_blank"><b>Download and read the National Library of Sweden’s national guidelines <i class="bi bi-box-arrow-up-right"></i></b></a>
+
+## How to share code & workflows?
+
+How you share code and workflows depends on your goals—whether you seek transparency, collaboration, or community contributions. Keep in mind that your objectives may evolve over time, and every step towards FAIR practices adds value.  
+
+<a class="link-teal" href="https://doi.org/10.17044/scilifelab.29086775.v1" target="_blank"> <b>Consult the SciLifeLab Open Science Software Checklist for comprehensive guidance<i class="bi bi-box-arrow-up-right"></i></b></a>
+
+### Where can code and workflows be shared?
+
+To ensure transparency, make a static copy of your code or workflow available for long-term access, independent of any research paper. This should be accompanied by a README file with a clear description, usage instructions, and a license. Collaborative version control platforms, such as GitHub, are ideal for both sharing and ongoing development. Use tags or releases to reference specific versions of your code.
```

Comment:
Regarding the license. For workflows there will be a license for the workflow itself, and then also for the tools it uses.